lms/static/sass/discussion/views/_thread.scss

```diff
@@ -106,24 +72,124 @@ body.discussion, .discussion-module {
     }
 
     .comment-actions-list {
+      @include float(right);
       display: inline-block;
       width: flex-grid(2,12);
       vertical-align: top;
     }
 
-    //TO-DO : clean up posted-details styling, currently reused by responses and comments
     .posted-details {
       margin-top: 0;
     }
   }
 }
 
+// +thread - wrapper styling
 .forum-thread-main-wrapper {
+  @include border-radius(3px, 3px, 0, 0);
   border-bottom: 1px solid $white; // Prevent collapsing margins
-  border-radius: 3px 3px 0 0;
   background-color: $white;
 }
 
+// +thread - elements - shared styles
+body.discussion {
+
+  .discussion-post, .discussion-response, .discussion-comment {
+    
+    // thread - images
+    .author-image {
+      @include margin-right($baseline/2);
+      display: inline-block;
+      background-color: $gray-l4;
+      vertical-align: top; 
+
+      // CASE: post image
+      &.level-post {
+        height: $post-image-dimension;
```

Comment:
Also, more of a user profile-based question, but can we always count on a user profile being square? Do we ask or give our users a heads up that images will be used in a square format?